Romiguier/Voleurs/src/Coffre.ts

```diff
@@ -0,0 +1,56 @@
+import type { EtatCoffre } from "./EtatCoffre";
+
+type CoffreParams = {
+  id: number;
```

Comment:
Do you really need an id? It doesn't look very useful for our use case.

Reply:
Nope :-) removed